internal/distro/distro_test.go

```diff
@@ -16,3 +23,41 @@ func TestDistro_Manifest(t *testing.T) {
 		distroregistry.NewDefault(),
 	)
 }
+
+func TestDistro_Version(t *testing.T) {
+	pipelinePath := "../../test/data/manifests/"
```

Comment:
I wonder if we want to base this test on the actual testing manifests. They change quite frequently and with the migration of 8.5 and 9.0 manifests to v2, this test will probably need a rewrite. Maybe we want to consider using a fixed mock manifest inlined in Go? I think we already have several of those.

Reply:
Yeah, that's definitely a better idea.

Reply:
Done!